### PR TITLE
rabbitmqadmin recipe

### PR DESCRIPTION
**Pull request by jjasghar**
This adds a recipe for [the management cli script](https://www.rabbitmq.com/management-cli.html).

### CODE REVIEW

**Comment by michaelklishin** [comment]
This makes me wonder if this cookbook should use the apt repo on Debian. This is something our packaging already handles: adding `rabbitmqadmin` to `PATH`.

**Comment by jjasghar** [comment]
@michaelklishin hmm, interesting. yeah the command never popped up; so i thought this was a way to just wrap it up and install it optionally.

File: recipes/rabbitmqadmin.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+#
+
+include_recipe 'python'
+
+remote_file '/usr/local/bin/rabbitmqadmin' do
+  source 'https://raw.githubusercontent.com/rabbitmq/rabbitmq-management/rabbitmq_v3_5_1/bin/rabbitmqadmin'
+  owner 'root'
+  group 'root'
+  mode '0644'
+  creates
```
Comment (cmluciano) [inline recipes/rabbitmqadmin.rb]: Is this line necessary?

File: recipes/rabbitmqadmin.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+include_recipe 'python'
+
+remote_file '/usr/local/bin/rabbitmqadmin' do
+  source 'https://raw.githubusercontent.com/rabbitmq/rabbitmq-management/rabbitmq_v3_5_1/bin/rabbitmqadmin'
```
Comment (cmluciano) [inline recipes/rabbitmqadmin.rb]: Should we just make this a template?

**Comment by johnbellone** [comment]
If you're interested I have a [rabbitmqadmin cookbook](https://github.com/johnbellone/rabbitmqadmin-cookbook) which has been working quite well.

**Comment by jjasghar** [comment]
I'm going to close this due to no activity, yes i dropped this completely. No one seems to have asked for this support either. :disappointed: :panda_face:

**Pull request closed without merging**